manifests/slave.pp

```diff
@@ -93,12 +93,13 @@
   $install_java             = $jenkins::params::install_java,
   $ensure                   = 'running',
   $enable                   = true
+  $maven_repo_url           = 'http://maven.jenkins-ci.org/content/repositories/releases',
```

Comment:
It's probably not important that the download path is actually a maven repo.  Would it be better to have a param to specify the complete url to the jar?